www/lib/ipfs.ts

```diff
@@ -1,3 +1,11 @@
+export function ipfsFetchableUrl(url: string) : string {
+  if (url.startsWith('ipfs://')) {
+    return 'https://infura-ipfs.io/ipfs/' + url.substring(7);
```

Comment:
why substring(7) seems like a magic number to me

Reply:
mostly because performance (I was looking at answers to https://stackoverflow.com/questions/9928679/remove-the-string-on-the-beginning-of-an-url); I was initially going to use a regex replacement.

but we could change that since it's a bit silly

Reply:
Ya for readability I'm changing this to a replace